pom.xml

```diff
@@ -143,18 +143,4 @@
       <url>https://repo.jenkins-ci.org/public/</url>
     </pluginRepository>
   </pluginRepositories>
-
-  <build>
```

Comment:
infuriating trying to read logs, can enable when needed, can tweak this before merging

Reply:
FWIW I typically set this mojo parameter to `FINE` and adjust log levels in the plugin accordingly.